cmd/network/show.go

```diff
@@ -195,6 +202,39 @@ var showCmd = &cobra.Command{
 					fmt.Println()
 				}
 				return
+			case selCommittee:
+				for _, runtime := range nodeStatus.Runtimes {
+					descriptor := runtime.Descriptor
+					if descriptor != nil {
+						runtimeID := descriptor.ID
+						paratimeName := getParatimeName(cfg, runtimeID.String())
+
+						fmt.Println("=== COMMITTEE ===")
+						fmt.Printf("Paratime: %s(%s)\n", paratimeName, runtimeID)
+						fmt.Printf("Height:   %d\n", height)
+						fmt.Println()
+						state, err := roothashConn.GetRuntimeState(ctx, &roothash.RuntimeRequest{
+							Height:    height,
+							RuntimeID: runtimeID,
+						})
+						cobra.CheckErr(err)
+
+						for i, member := range state.Committee.Members {
+							nodeQuery := &registry.IDQuery{
+								Height: height,
+								ID:     member.PublicKey,
+							}
+
+							node, err := consensusConn.Registry().GetNode(ctx, nodeQuery)
+							cobra.CheckErr(err)
+
+							fmt.Printf("%2d, entityID: %s, nodeID: %s, role: %s\n", i, node.EntityID, member.PublicKey.String(), member.Role.String())
```

Comment:
Could you output this in a table form? We already have support for outputting tables.